src/widget/text-print/text-print.js

```diff
@@ -12,32 +13,35 @@ class TextPrintWidget extends Widget {
     static get selector() {
         // The [data-for] exclusion prevents "comment" widgets from being included.
         // It is not quite correct to do this but atm the "for" attribute in XForms is only used for comment widgets.
-        return '.question:not(.or-appearance-autocomplete):not(.or-appearance-url) > input[type=text]:not(.ignore):not([data-for])';
+        return '.question:not(.or-appearance-autocomplete):not(.or-appearance-url) > input[type=text]:not(.ignore):not([data-for]), .question:not(.or-appearance-autocomplete):not(.or-appearance-url) > textarea:not(.ignore):not([data-for])';
     }
 
     _init() {
-        this.question.classList.add( 'or-text-print-initialized' );
+        this.question.classList.add( 'print-input-text' );
```

Comment:
I think we don't need any class here. We want to make sure that the real input fields still print if user bypass the print script (which is fine for non-Grid themes) by Cmd-P/Ctrl-P e.g.

Reply:
class removed